tests/test_iam.py

```diff
@@ -766,6 +766,120 @@ def test_iam_user_access_key_multi(self):
         self.assertEqual(
             resources[0]['c7n:matched-keys'][0]['c7n:match-type'], 'access')
 
+    def test_iam_user_ssh_key_filter(self):
```

Comment:
it would be nice to turn one of these into a terraform based test on setup to have it run in functional model

Reply:
Good call, I fleshed out the disable test a bit and converted that to terraform.